code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -142,57 +168,62 @@ Bonus
 			M.update_damage_overlays()
 
 	if(prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#FF3333")
+		new /obj/effect/temp_visual/heal(M.loc, "#FF3333")
 
 	return 1
 
 
 /*
 //////////////////////////////////////
 
-Flesh Mending
+Regenerative Coma
 
 	No resistance change.
 	Decreases stage speed.
 	Decreases transmittablity.
 	Fatal Level.
 
 Bonus
-	Heals brute damage over time. Turns cloneloss into burn damage.
+	Heals brute and burn while unconscious, but causes a long period of unconsciousness if the host is heavily damaged.
 
 //////////////////////////////////////
 */
 
-/datum/symptom/heal/brute/plus
-
-	name = "Flesh Mending"
-	desc = "The virus rapidly mutates into body cells, effectively allowing it to quickly fix the host's wounds."
+/datum/symptom/heal/coma
+	name = "Regenerative Coma"
+	desc = "The virus causes the host to fall into a coma when severely damaged, then rapidly fixes the damage."
 	stealth = 0
 	resistance = 0
 	stage_speed = -2
 	transmittable = -2
 	level = 8
 
-/datum/symptom/heal/brute/plus/Heal(mob/living/carbon/M, datum/disease/advance/A)
-	var/heal_amt = 4 * power
+/datum/symptom/heal/coma/CanHeal(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	if(M.IsUnconscious() || M.stat == UNCONSCIOUS)
+		return power
+	else if(M.stat == SOFT_CRIT)
+		return power * 0.5
+	else if(M.IsSleeping())
+		return power * 0.25
+	else if(M.getBruteLoss() + M.getFireLoss() >= 70)
+		to_chat(M, "<span class='warning'>You feel yourself slip into a regenerative coma...</span>")
+		M.Unconscious(450)

```

Comment:
This should probably be delayed and have a message so people have time to get somewhere safe or finish their current action.